packages/react/src/components/button.tsx

```diff
@@ -141,8 +140,8 @@ export function UploadButton<
   );
 
   const uploadFiles = useCallback(
-    (files: File[]) => {
-      void startUpload(files, fileRouteInput).catch((e) => {
+    async (files: File[]) => {
+      await startUpload(files, fileRouteInput).catch((e) => {
```

Comment:
why was this change made?

Reply:
I think it was just me debugging. reverting 👍

Reply:
![image](https://github.com/user-attachments/assets/53c2f034-9c79-4b66-92bd-d8800f7fb478)
nvm, it was because it doesn't pass lint